src/main/java/com/example/springbootintro/controller/CategoryController.java

```diff
@@ -0,0 +1,106 @@
+package com.example.springbootintro.controller;
+
+import com.example.springbootintro.dto.request.CategoryRequestDto;
+import com.example.springbootintro.dto.response.BookWithoutCategoryResponseDto;
+import com.example.springbootintro.dto.response.CategoryResponseDto;
+import com.example.springbootintro.service.BookService;
+import com.example.springbootintro.service.CategoryService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Category management", description = "Endpoints for managing books")
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("api/categories")
```

Comment:
```suggestion
@RequestMapping("/api/categories")
```
I guess you missed it